src/client/OpenSMT2/CMakeLists.txt

```diff
@@ -29,4 +29,4 @@ set_target_properties(solver_opensmt PROPERTIES
         COMPILE_FLAGS ${solver_opensmt_flags}
         )
 
-target_link_libraries(solver_opensmt client ${OPENSMT2_LIBRARY} gmp gmpxx z readline)
+target_link_libraries(solver_opensmt client ${OPENSMT2_LIBRARY} gmp gmpxx z ) #readline
```

Comment:
You can just remove readline here for now, maybe forever.

Reply:
Done!